sdk/test-utils/multi-version/README.md

```diff
@@ -0,0 +1,256 @@
+## Azure SDK for JavaScript's test-utils-multi-version
+
+The Azure SDK for JavaScript is composed of a multitude of repositories that attempt to deliver a
+common, homogenous SDK to make use of all of the services that Azure can provide. One of the
```

Comment:
```suggestion
common, homogenous SDK to make use of all of the services that Azure offers. One of the
```
?

Reply:
This was copied from test-utils-recorder...

Reply:
Ouch!

Reply:
Won't fix, to be consistent.